Build failure in fedora 21 / gcc 4.9.2

```
[  1%] /home/cjones/rr/obj/32/64bit_child.c: In function ‘main’:
/home/cjones/rr/obj/32/64bit_child.c:11:3: error: unknown type name ‘FILE’
   FILE* f = popen("echo -n", "r");
   ^
/home/cjones/rr/obj/32/64bit_child.c:11:3: error: implicit declaration of function ‘popen’ [-Werror=implicit-function-declaration]
/home/cjones/rr/obj/32/64bit_child.c:11:13: error: initialization makes pointer from integer without a cast [-Werror]
   FILE* f = popen("echo -n", "r");
             ^
/home/cjones/rr/obj/32/64bit_child.c:13:5: error: implicit declaration of function ‘fgetc’ [-Werror=implicit-function-declaration]
     int ch = fgetc(f);
     ^
/home/cjones/rr/obj/32/64bit_child.c:17:5: error: implicit declaration of function ‘putchar’ [-Werror=implicit-function-declaration]
     putchar(ch);
     ^
/home/cjones/rr/obj/32/64bit_child.c:19:3: error: implicit declaration of function ‘atomic_puts’ [-Werror=implicit-function-declaration]
   atomic_puts("EXIT-SUCCESS");
   ^
```

--- a/src/Command.h
+++ b/src/Command.h
@@ -3,6 +3,10 @@
 #ifndef RR_COMMAND_H_
 #define RR_COMMAND_H_
 
+#ifndef _DEFAULT_SOURCE
+# define _DEFAULT_SOURCE 1
+#endif
+
 #include <stdint.h>
 #include <stdio.h>
 

--- a/src/test/rrutil.h
+++ b/src/test/rrutil.h
@@ -4,6 +4,7 @@
 #define RRUTIL_H
 
 #define _GNU_SOURCE 1
+#define _POSIX_C_SOURCE 2
 
 #include <arpa/inet.h>
 #include <assert.h>

